.github/CODEOWNERS

```
@@ -0,0 +1 @@
+* @sonarsource/languages-team
```

This should be
```suggestion
.github/CODEOWNERS @sonarsource/languages-team
```
or
```suggestion
.github/CODEOWNERS @SonarSource/languages-team-dotnet
```

We'll put some GitHub actions here that are intended to be shared with other bubbles using GitHub Kanbas and backlog. I expect .NET Bubble to be the main contributor.